pkg/network/usm/sharedlibraries/watcher.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
-const (
+var (
```

Comment:
why did you change it?

Reply:
The test changes this to a much smaller value in order to avoid waiting for a long time.

Reply:
Can you leave a comment in the code then?